WooCommerce/UITestsFoundation/Screens/Orders/AddProductScreen.swift

```diff
@@ -21,4 +27,16 @@ public final class AddProductScreen: ScreenObject {
         app.buttons.staticTexts[name].tap()
         return try UnifiedOrderScreen()
     }
+
+    /// Selects multiple products from the list.
+    /// - Returns: Unified Order screen object.
+    @discardableResult
+    public func selectMultipleProducts(byName names: [String]) throws -> UnifiedOrderScreen {
+        app.buttons.staticTexts[names[0]].tap()
+        // TODO: Add a wait, and a second product
```

Comment:
This `TODO` will be done for M1.1 as per the conversation here p1677471329332259-slack-C02H2SVA1L5 . I'll remove the comment and add a link to the new GH issue before merging.